src/main/kotlin/com/workos/webhooks/models/UserUpdated.kt

```diff
@@ -14,63 +12,60 @@ open class UserUpdated
 @JsonCreator constructor(
   @JvmField
   @JsonProperty("object")
-  override val obj: String = "directory_user",
+  val obj: String = "directory_user",
 
   @JvmField
-  override val id: String,
+  val id: String,
 
   @JvmField
   @JsonProperty("directory_id")
-  override val directoryId: String,
+  val directoryId: String,
 
   @JvmField
   @JsonProperty("organization_id")
-  override val organizationId: String?,
+  val organizationId: String?,
 
   @JvmField
   @JsonProperty("idp_id")
-  override val idpId: String,
+  val idpId: String,
 
   @JvmField
   @JsonProperty("username")
-  override val userName: String?,
+  val userName: String?,
 
   @JvmField
   @JsonProperty("first_name")
-  override val firstName: String?,
+  val firstName: String?,
 
   @JvmField
   @JsonProperty("last_name")
-  override val lastName: String?,
+  val lastName: String?,
 
   @JvmField
   @JsonProperty("job_title")
-  override val jobTitle: String?,
+  val jobTitle: String?,
 
   @JvmField
   @JsonProperty("updated_at")
-  override val updatedAt: String,
+  val updatedAt: String,
 
   @JvmField
   @JsonProperty("created_at")
-  override val createdAt: String,
+  val createdAt: String,
 
   @JvmField
-  override val emails: List<Email>,
+  val emails: List<Email>,
 
   @JvmField
-  override val groups: List<Group>?,
-
-  @JvmField
-  override val state: UserState,
+  val state: UserState,
```

Comment:
I noticed a copy pasta error (not from this PR) that could use a fix here if you don't mind adding this as part of it

```suggestion
  val state: GroupState,
```